standalone/pherry/src/lib.rs

```diff
@@ -66,13 +66,13 @@ pub struct Args {
     )]
     no_sync: bool,
 
-    #[clap(long, help = "Don't write pRuntime egress data back to Substarte.")]
+    #[arg(long, help = "Don't write pRuntime egress data back to Substarte.")]
```

Comment:
Why don't all `#[clap` converted?

Reply:
This one is converted?

Reply:
I found some pherry `clap` not be converted, I guess that's because I revert Subxt upgrade by rollback pherry and phaxt changes